notifier/selfstate/selfstate.go

```diff
@@ -24,19 +28,25 @@ type SelfCheckWorker struct {
 	Notifier   notifier.Notifier
 	Config     Config
 	tomb       tomb.Tomb
-	Heartbeats []heartbeat.Heartbeater
+	heartbeats []heartbeat.Heartbeater
+}
+
+func NewSelfCheckWorker(logger moira.Logger, database moira.Database, notifier notifier.Notifier, config Config) *SelfCheckWorker {
```

Comment:
missed godoc and maybe we change order of variables? 

example: config first and logger last?

Reply:
I think godoc is not necessary, because it is constructor func (`New` prefix is a naming convention). We can write something about heartbeats, but idk.

Order of variables. I think it is ok to start with logger and db like `notifier.NewNotifier(database, logger, config, ...)` or `handler.NewHandler(database, logger, ...)` because we use logger and db everywhere and it is good to place it first like context.Context. It is more easy to add new parameters to the end

Reply:
by golang code-style all public func have to get godoc